gatsby-node.js

```diff
@@ -184,6 +187,13 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
           }
         }
       }
+      allIntegrations: allMdx(
+        filter: { fields: { collection: { eq: "integrations" } } }
+      ) {
+        nodes {
+          slug
```

Comment:
For knowledge, `slug` is a subfield of `fields`. So, this won't work.
Though, you should be removing this complete query and re-using the `allPosts` query fetched in the beginning and filtering the `integrations` collection just like other collections.

Reply:
done.